tests/unit/Exporter/ZipkinSpanConverterTest.php

```diff
@@ -32,8 +32,8 @@ public function shouldConvertASpanToAPayloadForZipkin()
         $this->assertSame($span->getSpanName(), $row['name']);
 
         $this->assertIsInt($row['timestamp']);
-        // timestamp should be in microseconds
-        $this->assertGreaterThan(1e15, $row['timestamp']);
+        // timestamp should be in nanoseconds
```

Comment:
pay attention that Zipkin needs timestamps in microseconds not nanoseconds. Also, if you want to test for nanoseconds it should be 1e18 not 1e12, because seconds timestamp already has 10 digits + another 9 to reach nanoseconds. For microseconds you should test against 1e15.
More info about the Zipkin Spans can be found here: https://zipkin.io/zipkin-api/#/definitions/Span